pkgs/development/python-modules/mkdocs-git-committers-plugin-2/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  # Evaluation
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonAtLeast,
+
+  # Dependencies
+  gitpython,
+  mkdocs,
+  requests,
+}:
+
+buildPythonPackage rec {
+  pname = "mkdocs-git-committers-plugin-2";
+  version = "2.3.0";
+  format = "setuptools";
+
+  disabled = ! pythonAtLeast "3.8";
+
+  src = fetchFromGitHub {
+    owner = "ojacques";
+    repo = "mkdocs-git-committers-plugin-2";
+    rev = "refs/tags/${version}";
+    hash = "sha256-+Ua2oX8PrfTROAhXNjcKdjIajVfvP3D3ToddFfj5N6A=";
+  };
+
+  propagatedBuildInputs = [
+    gitpython
+    mkdocs
+    requests
+  ];
+
+  pythonImportsCheck = [ "mkdocs_git_committers_plugin_2" ];
+
+  meta = with lib; {
+    description = "MkDocs plugin for displaying a list of contributors on each page";
+    homepage = "https://github.com/ojacques/mkdocs-git-committers-plugin-2";
+    changelog = "https://github.com/ojacques/mkdocs-git-committers-plugin-2/releases/tag/${version}";
+    license = licenses.mit;
+    maintainers = with maintainers; [ mahtaran ];
+  };
```

Comment:
```suggestion
  meta = {
    description = "MkDocs plugin for displaying a list of contributors on each page";
    homepage = "https://github.com/ojacques/mkdocs-git-committers-plugin-2";
    changelog = "https://github.com/ojacques/mkdocs-git-committers-plugin-2/releases/tag/${version}";
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [ mahtaran ];
  };
```

Reply:
Which of these two styles is preferred? A quick search seems to indicate that `with lib;` at the top level seems to be used more often, but perhaps this is a leftover from older preferences.

Reply:
`with lib;` is actively discouraged as far as I know.

Reply:
The reference manual also seems to prefer this style, so I'll do this instead!